src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -50,32 +51,312 @@ Notifier::~Notifier (void)
   LM_T(LmtNotImplemented, ("Notifier destructor is not implemented"));
 }
 
+
+
+/* ****************************************************************************
+*
+* templateNotify - 
+*
+* This function performs the necessary substitutions according to the template of
+* subscription to form the desired notification and send it to the endpoint specified
+* in the subscription.
+* 
+* 
+*/
+static bool templateNotify
+(
+  const SubscriptionId&            subscriptionId,
+  const ContextElement&            ce,
+  const ngsiv2::HttpInfo&          httpInfo,
+  const std::string&               tenant,
+  const std::string&               xauthToken,
+  const std::string&               fiwareCorrelator,
+  RenderFormat                     renderFormat,
+  const std::vector<std::string>&  attrsOrder
+)
+{
+  Verb                                verb = httpInfo.verb;
+  std::string                         method;
+  std::string                         url;
+  std::string                         payload;
+  std::map<std::string, std::string>  qs;
+  std::map<std::string, std::string>  headers;
+
+  LM_W(("KZ: In templateNotify"));
+  //
+  // 1. Verb/Method
+  //
+  if (verb == NOVERB)
+  {
+    // Default verb/method is POST
+    verb = POST;
+  }
+  method = verbName(verb);
+
+
+  //
+  // 2. URL
+  //
+  macroSubstitute(&url, httpInfo.url, ce);
+
+
+  //
+  // 3. Payload
+  //
+  if (httpInfo.payload == "")
+  {
+    payload = "{" + ce.toJson(renderFormat, attrsOrder) + "}";
+    LM_W(("KZ: payload: %s", payload.c_str()));
```

Comment:
This PR is 'temporal', not going to develop.
I am still using my traces, they will go away once 'definitive'